src/Kuzzle.js

```diff
@@ -198,6 +198,10 @@ function Kuzzle (host, options, cb) {
       value: undefined,
       enumerable: true,
       writable: true
+    },
+    sdkVersion: {
+      value: (typeof SDKVERSION === 'undefined') ? JSON.parse(require('fs').readFileSync('package.json', {encoding: 'utf-8'})).version : SDKVERSION,
```

Comment:
You can require a JSON file directly:

`value: (typeof SDKVERSION === 'undefined') ? require('../package.json').version : SDKVERSION`